dotcom-rendering/src/web/components/TopicFilterBank.tsx

```diff
@@ -62,19 +62,48 @@ const getKeyEventLink = (filterKeyEvents: boolean, id: Props['id']) => {
 	return `?${urlParams.toString()}#${id}`;
 };
 
-const isEqual = (selectedTopic: Topic, availableTopic: Topic) =>
-	availableTopic.type === selectedTopic.type &&
-	availableTopic.value === selectedTopic.value;
+const isEqual = (selectedTopic: Topic | undefined, availableTopic: Topic) => {
```

Comment:
I’m surprised we do not have a linting rule preventing this, but this is a confusing pattern, as one may consider omitting the first parameter.

I think it’s best to reduce the signature of this method to only accept a pair of `Topic`, and do the check of whether `selectedTopic` in `undefined` outside.

Reply:
Yesss makes sense, made the change 👍